src/codegen/mod.rs

```diff
@@ -90,6 +90,7 @@ pub struct Options {
     pub common_subexpression_elimination: bool,
     pub generate_debug_information: bool,
     pub opt_level: OptimizationLevel,
+    pub log_api_return_codes: bool,
 }
 
 impl Default for Options {
```

Comment:
`Options` is not a great name. I was thinking `CodegenOptions` but this is confusing because it also affects things in emit.

Reply:
I agree its confusing also because there are the opt levels in there. But would it be OK for you to have this refactored in another PR?

I can add that of my growing list of things I'd like to refactor probably soon anyways

Reply:
Of course, not sure what it should be called anyway. That was more of a note than review comment anyway